plugins/org.eclipse.emf.codegen/src/org/eclipse/emf/codegen/merge/java/facade/JRecord.java

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright (c) 2006 IBM Corporation and others.
```

Comment:
Did you really copy this from something or did you write this yourself.  If the latter, it should be your copyright.  (Often I just write "Eclipse contributors and  others."  If the former, 2006-2025 would better clarify that is revamped this year.

Reply:
I copied this from JType. I'm not sure how the copyright works for this project. I'd also be fine with Eclipse contributors and others.